python_scraper/python_scraper/pipelines.py

```diff
@@ -12,13 +12,23 @@
 
 
 class PythonScraperPipeline(object):
+
+    # Item - The item scraped
+    # Spider - The spider that scraped the item
     def process_item(self, item, spider):
-        if isinstance(item, DjangoItem):
-            instance = item.instance
-            commit = True
-            if isinstance(item, CompanyItem) and Company.objects.filter(series_lei=instance.series_lei,
-                                                                        rep_pd_date=instance.rep_pd_date).exists():
-                # skip items already in database, define series_lei and rep_pd_date to be unique
-                commit = False
-            item.save(commit=commit)
+        # Commit if:
+        #   Item is a Django Item
+        #   Item is not currently in the database
+        #       - series_lei and rep_pd_date act as primary keys
+        item.save(
```

Comment:
I think you are right with getting rid of the second if statement. The thing is we don't want item.save() to be called when its not a DjangoItem, other items could be included here as well, so I think we need the first one.

Reply:
My bad on this. Its been fixed now